bloom-housing/bloom
# 2165/rosefield preference updates

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2165)


## Description
This adds translations for the new rosefield listing preferences 

## How Can This Be Tested/Reviewed?
Run/tweak the following postgres script then with this checked out verify the following:

```
INSERT INTO preferences (Title, subtitle, description, links, form_metadata)
SELECT 
    'Previous Residents of Rosefield Village Relocated Outside of the City of Alameda' as Title,
    '' as subtitle,
    'At least one member of my household was a previous resident of Rosefield Village and could not be rehoused within the City of Alameda (assuming they meet the criteria for tenant selection and qualify for eligibility). ' as description,
    '[]' as links,
    '{"key":"rosefieldLive","options":[{"key":"yes","extraData":[]},{"key":"no","extraData":[]}]}' as form_metadata
;

INSERT INTO preferences (Title, subtitle, description, links, form_metadata)
SELECT 
    'Alameda Unified School District (AUSD) employee' as Title,
    '' as subtitle,
    'At least one member of my household is an Alameda Unified School District employee.At least one member of my household was a previous resident of Rosefield Village and could not be rehoused within the City of Alameda (assuming they meet the criteria for tenant selection and qualify for eligibility). ' as description,
    '[]' as links,
    '{"key":"rosefieldAUSD","options":[{"key":"yes","extraData":[]},{"key":"no","extraData":[]}]}' as form_metadata
;


SELECT
    *
FROM preferences
WHERE title = 'Previous Residents of Rosefield Village Relocated Outside of the City of Alameda'
;

SELECT
    *
FROM preferences
WHERE title = 'Alameda Unified School District (AUSD) employee'
;

SELECT
    *
FROM jurisdictions
WHERE name = 'Alameda'
;

INSERT INTO jurisdictions_preferences_preferences
SELECT 'b5d0ce66-160a-4b84-8118-1bd82668034c' as jurisdictions_id,
'e626a075-df15-47a4-a3f7-5bfbb2e1f918' as preferences_id
;

INSERT INTO jurisdictions_preferences_preferences
SELECT 'b5d0ce66-160a-4b84-8118-1bd82668034c' as jurisdictions_id,
'6a6c4311-13ff-4b22-b9a1-424b078934e7' as preferences_id
;
```

verify you can assign the new preferences to listing properly
verify the new preferences on a listing shows on public listing preview properly 
verify the new preferences on a listing shows properly on a public application form
verify the new preferences on a listing shows properly on a partner paper application form
verify a saved application has the preferences properly 
verify that csv export works with the new preferences

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
